src/plugins/hetero/tests/functional/shared_tests_instances/behavior/ov_infer_request/gpu_dyn_batch_shape_tests.cpp

```diff
@@ -0,0 +1,130 @@
+            result << ov::test::utils::partialShape2str({shape.first}) << "_";
+        }
+        result << "TS=";
+        for (const auto& shape : input_shapes) {
+            result << "(";
+            if (!shape.second.empty()) {
+                for (const auto& itr : shape.second) {
+                    result << ov::test::utils::vec2str(itr);
+                }
+            }
+            result << ")_";
+        }
+        result << "netPRC=" << model_type << "_";
+        result << "targetDevice=" << target_device;
+        if (!configuration.empty()) {
+            for (auto& configItem : configuration) {
+                result << "configItem=" << configItem.first << "_";
+                configItem.second.print(result);
+                result << "_";
+            }
+        }
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        if (core) {
+            core.reset();
+            core = ov::test::utils::PluginCache::get().core();
+        }
+        std::vector<InputShape> input_shape;
+
+        std::tie(input_shape, model_type, targetDevice, configuration) = this->GetParam();
+
+        init_input_shapes(input_shape);
+        // TODO: think how we can switch between several input topologies in the future
+        //   function = ov::test::utils::make_split_conv_concat(input_shape.front().first.get_min_shape(), model_type);
+        function = ov::test::utils::make_split_multi_conv_concat(input_shape.front().first.get_min_shape(), model_type);
+
+        //  make topology dynamic
+        std::map<std::string, ov::PartialShape> dynShape;
+        dynShape["input_tensor"] = input_shape.front().first;
+        function->reshape(dynShape);
+    }
+
+    ov::element::Type model_type;
+};
+
+TEST_P(OVDynamicBatchShape_Tests, InferDynamicBatchBound) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    run();
+}
+
+TEST_P(OVDynamicBatchShape_Tests, InferDynamicBatchBound_cached) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    std::string cacheFolderName;
+    {
+        std::stringstream ss;
+        ss << "InferDynamicBatchBound_cached_" << model_type << "_" << targetDevice;
+        cacheFolderName = ss.str();
+
+        ov::test::utils::removeFilesWithExt(cacheFolderName, "blob");
+        ov::test::utils::removeFilesWithExt(cacheFolderName, "cl_cache");
+        ov::test::utils::removeDir(cacheFolderName);
+
+        core->set_property(ov::cache_dir(cacheFolderName));
+        run();
+    }
+    {
+        core.reset();
+        core = ov::test::utils::PluginCache::get().core();
+        core->set_property(ov::cache_dir(cacheFolderName));
+        run();
+
+        ov::test::utils::removeFilesWithExt(cacheFolderName, "blob");
+        ov::test::utils::removeFilesWithExt(cacheFolderName, "cl_cache");
+        ov::test::utils::removeDir(cacheFolderName);
+    }
+}
+
+auto hetero_config = []() {
+    return ov::AnyMap{{ov::device::priorities.name(), ov::test::utils::DEVICE_GPU}};
+};
```

Comment:
```suggestion
const auto hetere_config = testing::Values(ov::AnyMap{{ov::device::priorities.name(), ov::test::utils::DEVICE_GPU}});
```
Can use similar approach in other test parameters, instead make vectors copy to values.